packer/run-sca-jenkins-node.sh

```diff
@@ -0,0 +1,95 @@
+#!/bin/bash
+set -e
+
+# ##########################################
```

Comment:
I think you can keep the SCA setup as a separate script from the registration of the Jenkins slave. That will give you move flexibility in how you launch the instance. I would just have the user-data script run the sca setup script and then run the separate Jenkins slave script.